### PR TITLE
U Update the document of  `sendSignedTransaction`.

### LINKED ISSUE #2915: sendSignedTransaction() error: invalid sender
#### Expected behavior

    Calling `web3.eth.sendSignedTransaction(data)` should succeed.

#### Actual behavior

    Calling `web3.eth.sendSignedTransaction(data)` 
    returns `error: {"code":-32000,"message":"invalid sender"}`.

#### Steps to reproduce the behavior

```javascript
const Tx = require('ethereumjs-tx');
const privateKey = new Buffer('e33xxxxc0fd109', 'hex')

const rawTx = {
  nonce: '0x00',
  gasPrice: '0x09184e72a000',
  gasLimit: '0x2710',
  to: '0x0000000000000000000000000000000000000000',
  value: '0x00',
  data: '0x7f7465737432000000000000000000000000000000000000000000000000000000600057'
}

const tx = new Tx(rawTx);
tx.sign(privateKey);

const serializedTx = tx.serialize();

web3.eth.sendSignedTransaction('0x' + serializedTx.toString('hex'))
.on('receipt', console.log);
```

#### Error Logs 
```
error: {"code":-32000,"message":"invalid sender"}
```

### Versions 
* web3.js: 1.0.0-beta.55
* nodejs: v11.10.1
* browser: Chrome

### PR DESCRIPTION
**Pull request by Skyge**
Fix #2915
Because the package `ethereumjs-tx` has upgraded to the version `2.0.0`, and it make some changes, so we need to make corresponding changes.

### CODE REVIEW

File: docs/web3-eth.rst [hunk 1]
```diff
@@ -1096,8 +1096,8 @@ Example
 
 .. code-block:: javascript
 
-    var Tx = require('ethereumjs-tx');
-    var privateKey = new Buffer('e331b6d69882b4cb4ea581d88e0b604039a3de5967688d3dcffdd2270c0fd109', 'hex')
+    var Tx = require('ethereumjs-tx').Transaction;
+    var privateKey = new Buffer.from('e331b6d69882b4cb4ea581d88e0b604039a3de5967688d3dcffdd2270c0fd109', 'hex')
```
Comment (shumailxyz) [inline docs/web3-eth.rst]: You don't need `new` here. `Buffer.from(..)` should work.

Comment (RjtSinghal) [reply docs/web3-eth.rst]: i am still getting the error. 
error: {"code":-32000,"message":"invalid sender"}

Comment (RjtSinghal) [reply docs/web3-eth.rst]: issue fix. I just specified 'chain'  parameter while creating transaction and it works.
const tx = new Tx(txObject, {'chain':'ropsten'})

Comment (Skyge) [reply docs/web3-eth.rst]: > You don't need `new` here. `Buffer.from(..)` should work.

@shumailxyz Maybe, I will have a try later.

**Comment by nivida** [comment]
Could you open a second PR with these documentation changes for the 2.x branch? Thanks!:)

**Review by nivida — APPROVED** [review]

**Comment by Skyge** [comment]
@nivida OK, I will do it now for the 2.x branch.

**Comment by Skyge** [comment]
By the way, I am not sure what is the difference between the branch `1.x` and `2.x`. 
And what about the branch `0.20.7`, should I also make a same document PR?

**Pull request merged**